content/docs/command-reference/params/index.md

```diff
@@ -16,62 +16,70 @@ positional arguments:
 
 ## Description
 
-In order to track parameters and hyperparameters associated to machine learning
-experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. They usually have simple names like `epochs`,
-`learning-rate`, `batch_size`, etc.
+Parameters are simple values used inside your code to influence the results
+(e.g. machine learning [hyperparameters]). DVC can track these as key/value
+pairs from structured YAML 1.2, JSON, TOML 1.0,
+[or Python](#examples-python-parameters-file) files (`params.yaml` by default),
+from which your code should also read them.
 
-To start tracking parameters, list them under the `params` field of `dvc.yaml`
-stages (manually or with the the `-p`/`--params` option of `dvc stage add`). For
-example:
+Multiple stages of a <abbr>pipeline</abbr> can [use the same params file] as
+<abbr>dependency</abbr>, but only certain values will affect each
+<abbr>stage</abbr>. Params usually have simple names like `epochs`,
+`learning-rate`, `batch_size`, etc. Example:
```

Comment:
```
Params usually have simple names like `epochs`,
`learning-rate`, `batch_size`, etc.
```

Minor but IMO we don't need this sentence (or I'm not understanding the point of it).

Reply:
I remember Dmitry added this to the reference to give a quick idea of what we're talking about (example-based).

Reply:
p.s. OK I reorganized the text so this is in the first paragraph (again) and moved the params.yaml sample up too.